Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/EditorModeFeedbackFeatureProcessor.cpp

```diff
@@ -143,5 +140,19 @@ namespace AZ
 
             m_editorStatePassSystem->Update();
         }
+
+        void EditorModeFeatureProcessor::OnTick(float, AZ::ScriptTimePoint)
+        {
+            // Attempt deferred loading of mask material until the asset is ready
+            if (!m_maskMaterial)
+            {
+                m_maskMaterial = CreateMaskMaterial();
```

Comment:
Could be expensive to request this on every tick, but this performance concern can be addressed after the crash is removed.

Reply:
So luckily this will only happen for a short period of time and then when `m_maskMaterial` is set, we disconnect from the tick bus and don't have to update anymore, so this should be an incredibly minimal performance overhead thankfully